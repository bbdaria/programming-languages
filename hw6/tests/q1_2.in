use_module(library(yall)).
set_prolog_flag(answer_write_options, [quoted(true), portray(true), max_depth(0), spacing(next_argument)]).
print("===TEST START===").

{L}/(
    findall(
        [X],
        merge_bt(bt(3, [bt(4, [])]), bt(5, [bt(7, [])]), X),
        L1
    ),
    msort(L1, L)
).

{L}/(
    findall(
        [X],
        merge_bt(bt(1, []), bt(5, []), X),
        L1
    ),
    msort(L1, L)
).