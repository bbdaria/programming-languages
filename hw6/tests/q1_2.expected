true.

L = [[bt(3, [bt(5, [bt(7, [])]), bt(4, [])])]].


L = [[bt(1, [bt(5, [])])]].


